src/Lazy.elm

```diff
@@ -17,7 +17,7 @@ module Lazy
 @docs apply, andThen
 -}
 
-import Native.Lazy
+import Native.Lazy exposing (..)
```

Comment:
This should be reverted. Since the variables all live in JS, it's impossible to know what to expose here. So all native things need to be fully namespaced.